### PR TITLE
Add ocs deployment in UI on lso

### PR DESCRIPTION
**Pull request by ebondare**
Make changes to ocs operation installation function to make it re-usable for local storage operator as well; add a function for the creation of storage cluster in "internal - attached devices" mode

### CODE REVIEW

File: ocs_ci/ocs/ui/deployment_ui.py [hunk 1]
```diff
@@ -154,24 +154,29 @@ def create_catalog_source_yaml(self):
...
-            self.dep_loc["search_operators"], text="OpenShift Container Storage"
-        )
+        logger.info(f"Search {operator_name} Operator")
+        self.do_send_keys(self.dep_loc["search_operators"], text=operator_name)
 
-        logger.info("Choose OCS Version")
-        self.do_click(self.dep_loc["choose_ocs_version"])
+        logger.info(f"Choose {operator_name} Version")
+        if operator_name == "OpenShift Container Storage":
+            self.do_click(self.dep_loc["choose_ocs_version"])
```
Comment (ebenahar) [inline ocs_ci/ocs/ui/deployment_ui.py:169]: consider passing to the function the actual clickable name of the operator, such as `choose_ocs_version`. This way you will not have to put these conditions here.

You can also save `choose_ocs_version` as a constant, something like `UI_OCS_OPERATOR_CLICK`

**Comment by OdedViner** [comment]
Need to add local storage operator 4.7:
```
oc create -f https://raw.githubusercontent.com/red-hat-storage/ocs-ci/master/ocs_ci/templates/ocs-deployment/local-storage-optional-operators.yaml
oc patch image.config.openshift.io/cluster --type merge -p '{"spec": {"registrySources": {"insecureRegistries": ["brew.registry.redhat.io", "registry-proxy.engineering.redhat.com", "registry.stage.redhat.io"]}}}'


$ oc get mcp
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-db483ee5106c0985e126eef0958a8008   False     True       False      3              1                   1                     0                      2d16h
worker   rendered-worker-519cae155dabdb5d7db357225bafcc37   False     True       False      6              1                   1                     0                      2d16h

Wait 10 min

$ oc get mcp
NAME     CONFIG                                             UPDATED   UPDATING   DEGRADED   MACHINECOUNT   READYMACHINECOUNT   UPDATEDMACHINECOUNT   DEGRADEDMACHINECOUNT   AGE
master   rendered-master-122b032fb96ff7a3090414cbc93532f6   True      False      False      3              3                   3                     0                      2d16h
worker   rendered-worker-effcc54318aaf13f7d27aadae8b3089b   True      False      False      6              6                   6                     0                      2d16h

```

File: ocs_ci/ocs/ui/deployment_ui.py [hunk 2]
```diff
@@ -238,6 +245,44 @@ def install_internal_cluster(self):
...
+        self.do_click(locator=self.dep_loc["yes_button"])
+
+        logger.info("Sleep for 60 seconds to wait for nodes to be discovered")
+        time.sleep(60)
+
+        logger.info("Go to Step 4")
+        self.do_click(locator=self.dep_loc["submit_button"])
+
+        # Encryption is disabled by default. TODO: add encryption
+
```
Comment (OdedViner) [inline ocs_ci/ocs/ui/deployment_ui.py:276]: @ebondare Can use this configure_encryption function on lso?
https://github.com/red-hat-storage/ocs-ci/blob/afd9e58b16b8729f9fe04c76a5e08b0d285a4403/ocs_ci/ocs/ui/deployment_ui.py#L241

**Comment by petr-balogh** [comment]
I think it requires rework on this PR cause of conflicts and there is no activity for long time.
I am closing this PR, please feel free to rebase, fix and re-open the PR or open the new one.
Thanks

**Pull request closed without merging**